yarn.lock

```diff
@@ -7158,23 +7158,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"glob@npm:^10.2.2, glob@npm:^10.3.10, glob@npm:^10.3.7":
```

Comment:
Those 3 ranges correspond to the 3 `yarn` commands I had to run (see PR description)